indico_hub/models.py

```diff
@@ -1,10 +1,32 @@
 import uuid
 
+from flask import json
 from sqlalchemy.dialects.postgresql import UUID
+from sqlalchemy.types import VARCHAR, TypeDecorator
 
 from .db import db
 
 
+class JSONEncodedDict(TypeDecorator):
```

Comment:
Why not use the native JSONB type from SQLAlchemy?

Reply:
solved